app/forms/ApplicationContactDetailsFormProvider.scala

```diff
@@ -41,8 +41,13 @@ class ApplicationContactDetailsFormProvider @Inject() extends Mappings {
           .verifying(maxLength(Validation.emailMaxLength, emailLengthError))
           .verifying(Constraints.pattern(Validation.emailPattern, error = emailFormatError)),
         "phone" -> text(phoneRequiredError)
-          .verifying(phoneFormatError, isValid(_))
-          .verifying(maxLength(Validation.phoneNumberMaxLength, phoneLengthError))
+          .verifying(
+            phoneFormatError,
+            phone =>
+              isValid(phone)
+                && phone.length <= Validation.phoneNumberMaxLength
+                && !phone.exists(_.isLetter)
+          )
```

Comment:
Google phone lib allows letters in phone numbers (think KL5-3226). I've seen this crop up on another project using this library 😞

This will continue to allow formatting `-`, ` `, `(`, `+`, etc whilst stopping such numbers

Reply:
Seems the version of the library we are using is a year old. The newest version 8.13.10 was released a couple of days ago. Although that's not to say the issue is fixed in the newest version.